brain/compartmentReportReader.cpp

```diff
@@ -120,7 +124,7 @@ const CompartmentReportMapping& CompartmentReportView::getMapping() const
 }
 
 std::future<CompartmentReportFrame> CompartmentReportView::load(
-    const float timestamp)
+    double timestamp)
```

Comment:
This males the code inconsistent with load(double start, double end) and with the method declaration signature (which also makes my code editor not working properly).